docs/catalyst_101.md

```diff
@@ -122,7 +122,7 @@ Once the existing members have voted to offer you membership, it’s up to you t
 #### Accepting Membership
 
 - Sign the [Membership Agreement](https://drive.google.com/file/d/1tt4IXVwKi2Gj-L755AUZyu1QaVDRO8px/view?usp=sharing)
-- Purchase $1000 share, your initial capital contribution to the co-op, by sending Steve a check in the mail.
+- Purchase $1000 share, your initial capital contribution to the co-op, by sending us a check in the mail.
```

Comment:
Who do we make the check out to, and what address are we using now?

Reply:
I'm not sure...I tagged CG in slack about this

Reply:
But I agree we should include information here rather than just saying ~send us a check~

Reply:
Sounds good. I'll approve once @cmgosnell gets back to you.

Reply:
Ok CG says to send it to her, I'll update this

Reply:
Does it have to be a check? If someone can do an ACH transfer would we not want to do that instead?

Reply:
I can text Steve about this

Reply:
Apparently we can use ACH but there is a fee of about $10-15